packages/driver/src/cypress/log.ts

```diff
@@ -181,8 +177,10 @@ const defaults = function (state, config, obj) {
     return t._currentRetry || 0
   }
 
+  counter++
+
   _.defaults(obj, {
-    id: (counter += 1),
+    id: `log-${window.location.origin}-${counter}`,
```

Comment:
Does this correct the issue reported in https://github.com/cypress-io/cypress/issues/19545?

Reply:
No, unfortunately not.